File: src/main/java/com/yuriysurzhikov/gidassistant/utils/DateUtils.java
package com.yuriysurzhikov.gidassistant.utils;
import java.util.*;

public class DateUtils {
    public static Integer calculateAge(Long birthday) {
        Date bd = Calendar.getInstance().getTime();
        bd.setTime(birthday);
        return Calendar.getInstance().getTime().getYear() - bd.getYear();
    }
}

File: src/main/java/com/yuriysurzhikov/gidassistant/utils/DateUtils.kt
package com.yuriysurzhikov.gidassistant.utils

import java.text.SimpleDateFormat
import java.util.*

object DateUtils {
    @JvmStatic
    fun calculateAge(birthday: Long): Int? {
        val bd = Calendar.getInstance().time
        bd.time = birthday
        return Calendar.getInstance().time.year - bd.year
    }
}